### ISSUE TITLE
ikfast61_moveit_plugin_template.cpp not installed

### ISSUE THREAD

**Issue by bit-pirate** [moveit/moveit_ikfast#17]
```
$ rosrun moveit_ikfast create_ikfast_moveit_plugin.py korus arm korus_moveit_ikfast_plugin /home/marcus/.openrave/kinematics.e949180cc0c529f8201273314f70614a/ikfast61.TranslationZAxisAngle4D.0_2_3_4_f1.cpp 

IKFast Plugin Generator
Loading robot from 'korus_moveit_config' package ... 
Creating plugin in 'korus_moveit_ikfast_plugin' package ... 
  found 3 planning groups: arm, gripper, head
  found group 'arm'
  found source code generated by IKFast version 61

ERROR: can't find template file at '/opt/ros/hydro/share/moveit_ikfast/templates/ikfast61_moveit_plugin_template.cpp'
```

**Comment by davetcoleman** [comment]
Is this still an issue?

**Comment by gavanderhoorn** [comment]
I think this was fixed in 2c15939e.

Closing. @bit-pirate: if you feel this has not been fixed, please reopen and provide additional information.
